src/components/app/UrlManager.js

```diff
@@ -106,7 +106,11 @@ class UrlManagerImpl extends React.PureComponent<Props> {
     // will be done for all URLs.
     let browserConnectionStatus;
     const route = window.location.pathname.split('/').filter((s) => s)[0];
-    if (['from-browser', 'from-addon', 'from-file'].includes(route)) {
+    if (
+      ['from-browser', 'from-addon', 'from-file', 'from-post-message'].includes(
+        route
+      )
+    ) {
```

Comment:
Probably not needed? I'm not sure though.
Actually I wonder if we shouldn't have a browser connection for all data sources actually...
(yeah, I'm giving contradictory comments in the same sentence)

Reply:
I wasn't really sure either, so it seemed safer to add it.